implants/lib/eldritch/src/file/temp_file_impl.rs

```diff
@@ -0,0 +1,79 @@
+use anyhow::Result;
+use std::env;
+use std::fs::File;
+use tempfile::NamedTempFile;
+
+pub fn temp_file(name: Option<String>) -> Result<String> {
+    let mut temp_path;
+    let _file;
```

Comment:
Optional nit : if we're not using this bar can we remove it and instead replace refs with _

Reply:
i think i need to leave this declared outside the match. or maybe i misunderstanding the ask here

Reply:
No worries: instead of _file for the left side of the operation you can use _ as a throwaway.

So you can get rid of the variable _file and replace it on later lines with _

Reply:
didnt know this. will update quick